src/Assets.php

```diff
@@ -81,12 +81,15 @@ public static function register_assets() {
 		self::register_script( 'wc-price-filter', plugins_url( self::get_block_asset_build_path( 'price-filter' ), __DIR__ ), $block_dependencies );
 		self::register_script( 'wc-attribute-filter', plugins_url( self::get_block_asset_build_path( 'attribute-filter' ), __DIR__ ), $block_dependencies );
 		self::register_script( 'wc-active-filters', plugins_url( self::get_block_asset_build_path( 'active-filters' ), __DIR__ ), $block_dependencies );
-		self::register_script( 'wc-checkout-block', plugins_url( self::get_block_asset_build_path( 'checkout' ), __DIR__ ), $block_dependencies );
-		self::register_script( 'wc-cart-block', plugins_url( self::get_block_asset_build_path( 'cart' ), __DIR__ ), $block_dependencies );
 
-		if ( 'experimental' === WOOCOMMERCE_BLOCKS_PHASE ) {
+		if ( WOOCOMMERCE_BLOCKS_PHASE > 2 ) {
```

Comment:
Again a corresponding PHP utility function might be useful. It's also clearer to read code if the phases are named - but you don't need to remove the numbered phases from behind the scenes, they are fine.